django_project/lesson/views/worksheet.py

```diff
@@ -257,3 +260,50 @@ def post(self, request, *args, **kwargs):
         section = Section.objects.get(slug=kwargs.get('section_slug'))
         worksheets = Worksheet.objects.filter(section=section)
         return re_order_features(request, worksheets)
+
+
+
+class WorksheetModuleQuestionAnswers(WorksheetMixin, SectionMixin, DetailView):
+    """Show correct answers to module questions.
+
+    :param request: HttpRequest object
+    :type request: HttpRequest
+    """
+    context_object_name = 'worksheets'
+    template_name = 'worksheet/question_answers.html'
+
+    def get_context_data(self, **kwargs):
+        """Create context for use in the templates."""
+
+        context = super(WorksheetModuleQuestionAnswers, self).get_context_data(
+         **kwargs)
+        project_slug = self.kwargs.get('project_slug', None)
+        section_slug = self.kwargs.get('section_slug', None)
+        project = get_object_or_404(Project, slug=project_slug)
+
+        context['sections'] = Section.objects.filter(project=project,
+                                                     slug=section_slug)
+        for section in context['sections']:
+            query_set = Worksheet.objects.filter(section=section)
+            context['worksheets'] = []
+
+            for worksheet in query_set:
+                worksheet_json = {'worksheet': worksheet,
+                                   'question_answers':[]
```

Comment:
__[Flake8]__

[E127] continuation line over-indented for visual indent


<!-- Comment by [SideCI](https://sideci.com) -->